src/runtime/config.ts

```diff
@@ -21,6 +21,12 @@ const mergeWithEnvVariables = createDefu((obj: Record<string, any>, key: string,
     obj[key] = override
     return true
   }
+
+  // Avoid duplicating arrays content (default defu behavior)
```

Comment:
Defu just concatenates. It is due to our workaround use that runtime config is contaminated it itself (https://github.com/unjs/nitro/blob/10d48a64229bc3bda9d23428c5f057760339bc0d/src/runtime/config.ts#L33)